libopenage/curve/continuous.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017-2017 the openage authors. See copying.md for legal info.
+
+#pragma once
+
+#include "internal/value_container.h"
+
+#include "../log/log.h"
+
+namespace openage {
+namespace curve {
+
+/**
+ * Continuous Datatype.
+ * Stores a value container with continuous access.
+ * The bound template type _T has to implement `operator+(_T)` and
+ * `operator*(curve_time_t)`.
+ *
+ * pxd:
+ * cppclass Continuous(ValueContainer):
+ *     _T get(const curve_time_t&) except +
+ */
+template<typename _T>
```

Comment:
We should agree on a project wide style for naming template parameters. `_T` is new to me, maybe we could do that everywhere. Needs discussion.